checkov/cdk/checks/typescript/IAMPolicyAttachedToGroupOrRoles.yaml

```
@@ -0,0 +1,28 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_40
+  name: Ensure IAM policies are attached only to groups or roles (Reducing access management complexity may in-turn reduce opportunity for a principal to inadvertently receive or retain excessive privileges.)
+  category: IAM
+  framework: cdk
+scope:
+  languages:
+    - typescript
+definition:
+  or:
+    - pattern: |
+        new $PACKAGE.aws_iam.Policy(<ANY>, <ANY>, { <ANY>, users: <ANY>, <ANY> })
```

```suggestion
        new $PACKAGE.aws_iam.Policy(<ANY>,  { <ANY>, users: <ANY>, <ANY> })
```
Check if the ANY that is outer to dict is redundant. If not - PLS open a ticket to add it in the Wrapper so patterns writing will be simplified.